tool/tctl/common/resource_command_test.go

```diff
@@ -102,9 +119,12 @@ func TestDatabaseServerResource(t *testing.T) {
 	}
 
 	db1, err := types.NewDatabaseV3(types.Metadata{
-		Name:        "example",
+		Name:        "example-rds-us-west-1",
 		Description: "Example MySQL",
-		Labels:      map[string]string{types.OriginLabel: types.OriginConfigFile},
+		Labels: map[string]string{
+			types.OriginLabel:         types.OriginConfigFile,
```

Comment:
nit:
```suggestion
			types.OriginLabel:         types.OriginCloud,
```
same for `"example-rds-us-west-2"`

Reply:
it gets set by teleport init to config file